server.py

```diff
@@ -39,7 +39,17 @@ def dump_table():
         return make_response(jsonify('Table name not supplied.'), 400)
     try:
         # TODO: once authentication is in place, restrict the tables that can be listed here
-        table_info_obj = driver.get_table(table_name)
+        columns = request.args.get('column', '')
+        if columns == '':
+            columns = None
+        else:
+            app.logger.info('columns')
```

Comment:
Is there a reason to keep these logging statements?

Reply:
Not my code, I don't know who wrote it.

Reply:
Oh, that is for logging the arguments, keep there for debugging, it doesn't matter.